packages/mui-base/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/base",
-  "version": "5.0.0-beta.60",
+  "version": "5.0.0-beta.61",
```

Comment:
Do we still want to release @mui/base? cc @mnajdova

Reply:
Yep, let's release it.